msal/tests/dev apps/XForms/XForms/App.xaml.cs

```diff
@@ -39,16 +39,17 @@ public partial class App : Application
         public static PublicClientApplication MsalPublicClient;
         public static UIParent UIParent { get; set; }
         public const string DefaultClientId = "4b0db8c2-9f26-4417-8bde-3f0e3656f8e0";
-        public const string B2cClientId = "750a8822-a6d4-4127-bc0b-efbfacccbc28";
+        public const string B2cClientId = "e3b9ad76-9763-4827-b088-80c7a7888f79";
 
         public const string RedirectUriOnIos = "adaliosxformsapp://com.yourcompany.xformsapp";
-        public const string RedirectUriOnAndroid = "urn:ietf:wg:oauth:2.0:oob";
+        public static string RedirectUriOnAndroid = "urn:ietf:wg:oauth:2.0:oob";
```

Comment:
Why switch to static?

Reply:
we need to change the redirectUri for b2c, so i'm setting that in the settings page (where the b2c switch is located), which means this value now needs to be static because it's no longer constant. 

---
In reply to: [233221288](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1403#discussion_r233221288) [](ancestors = 233221288)